openfl/display/DrawCommandReader.hx

```diff
@@ -0,0 +1,434 @@
+	public function readCurveTo           ():CurveToView           { advance (); prev = CURVE_TO;            return new CurveToView           (this); }
+	public function readDrawCircle        ():DrawCircleView        { advance (); prev = DRAW_CIRCLE;         return new DrawCircleView        (this); }
+	public function readDrawEllipse       ():DrawEllipseView       { advance (); prev = DRAW_ELLIPSE;        return new DrawEllipseView       (this); }
+	public function readDrawRect          ():DrawRectView          { advance (); prev = DRAW_RECT;           return new DrawRectView          (this); }
+	public function readDrawRoundRect     ():DrawRoundRectView     { advance (); prev = DRAW_ROUND_RECT;     return new DrawRoundRectView     (this); }
+	public function readDrawTiles         ():DrawTilesView         { advance (); prev = DRAW_TILES;          return new DrawTilesView         (this); }
+	public function readDrawTriangles     ():DrawTrianglesView     { advance (); prev = DRAW_TRIANGLES;      return new DrawTrianglesView     (this); }
+	public function readEndFill           ():EndFillView           { advance (); prev = END_FILL;            return new EndFillView           (this); }
+	public function readLineStyle         ():LineStyleView         { advance (); prev = LINE_STYLE;          return new LineStyleView         (this); }
+	public function readLineBitmapStyle   ():LineBitmapStyleView   { advance (); prev = LINE_BITMAP_STYLE;   return new LineBitmapStyleView   (this); }
+	public function readLineGradientStyle ():LineGradientStyleView { advance (); prev = LINE_GRADIENT_STYLE; return new LineGradientStyleView (this); }
+	public function readLineTo            ():LineToView            { advance (); prev = LINE_TO;             return new LineToView            (this); }
+	public function readMoveTo            ():MoveToView            { advance (); prev = MOVE_TO;             return new MoveToView            (this); }
+	public function readDrawPathC         ():DrawPathCView         { advance (); prev = DRAW_PATH_C;         return new DrawPathCView         (this); }
+	public function readOverrideMatrix    ():OverrideMatrixView    { advance (); prev = OVERRIDE_MATRIX;     return new OverrideMatrixView    (this); }
+	
+	public inline function tileSheet (index:Int):Tilesheet {
+		
+		return buffer.ts[tsPos + index];
+		
+	}
+	
+	public inline function fArr (index:Int):Array<Float> {
+		
+		return buffer.ff[ffPos + index];
+		
+	}
+	
+	public inline function iArr (index:Int):Array<Int> {
+		
+		return buffer.ii[iiPos + index];
+		
+	}
+	
+	public inline function float (index:Int):Float {
+		
+		return buffer.f[fPos + index];
+		
+	}
+	
+	public inline function int (index:Int):Int {
+		
+		return buffer.i[iPos + index];
+		
+	}
+	
+	public inline function bool (index:Int):Bool {
+		
+		return buffer.b[bPos + index];
+		
+	}
+	
+	public inline function obj (index:Int):Dynamic {
+		
+		return buffer.o[oPos + index];
+		
+	}
+
+}
+
+@:forward
```

Comment:
I think so, it wasn't compiling without it and someone suggested it and then it worked. TBH I have no idea what it means or does.